recipes/databricks-cli/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "databricks-cli" %}
+{% set version = "0.8.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9db04827ded11313892b7f190f95af00677b91743c4b7009e1094f7f1ffc6bc7
+
+build:
+  number: 0
+  entry_points:
+    - databricks=databricks_cli.cli:cli
+    - dbfs=databricks_cli.dbfs.cli:dbfs_group
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - click >=6.7
+    - configparser >=0.3.5
+    - python
+    - requests >=2.17.3
+    - setuptools
```

Comment:
Hey @jaroslawk, why do we need setuptools here?

Reply:
it's not needed - removing it.